### ISSUE TITLE
Antidote syringe.

### ISSUE THREAD

**Issue by Afavertov** [studio-minus/ppg-bugs-and-requests#537]
Syringe that will clean human blood from every negative liquid.

Save human from toxins or cure from virus.

**Comment by DindinYT37** [comment]
So basically... a Life Syringe?

**Comment by Afavertov** [comment]
Even if you inject life syringe in human,, he will continue dissolving by acid.

**Comment by DindinYT37** [comment]
> Even if you inject life syringe in human,, he will continue dissolving by acid.

Of course, it's not supposed to be op. Also if there were a syringe that would be able to stop the acid, it would be useless since the acid is pretty fast and you would've already lost a few limbs or even died by the time, the syringe would be injected into the Human.

**Comment by TamasB14** [comment]
I'd still use something like that. A simple "cleanser" syringe that doesn't heal, but removes all effects (not just negative ones). Eliminates Zombie virus, Super Strength, Acid and such things.

**Comment by DindinYT37** [comment]
> I'd still use something like that. A simple "cleanser" syringe that doesn't heal, but removes all effects (not just negative ones). Eliminates Zombie virus, Super Strength, Acid and such things.

I mean we could use that as feature, but that already exists as a mod.

**Comment by idotwaffle** [comment]
The reconstructor removes all foreign liquids!

**Comment by Afavertov** [comment]
Reconstructor cant heal zombie virus.
